services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -699,6 +703,30 @@ private void doDeleteMeter(CommandMessage message, String replyToTopic, Destinat
         }
     }
 
+    private void doConfigurePort(final CommandMessage message, final String replyToTopic, 
+            final Destination replyDestination) {
+        PortConfigurationRequest request = (PortConfigurationRequest) message.getData();
+        
+        try {
+            logger.info("Port configuration request. Switch '{}', Port '{}'", request.getSwitchId(), 
+                    request.getPortNumber());
+            ISwitchManager switchManager = context.getSwitchManager();
+            PortConfigurationResponse response = switchManager.configurePort(request);
+
+            InfoMessage infoMessage = new InfoMessage(response, message.getTimestamp(),
+                    message.getCorrelationId());
+            context.getKafkaProducer().postMessage(replyToTopic, infoMessage);
+        } catch (Exception e) {
+            logger.error("Port configuration request failed. Switch '{}', Port '{}'", request.getSwitchId(), 
```

Comment:
Any details why it failed?

Reply:
When the switch or port was not found.

Reply:
@ashishaggarwalCS, @surabujin meant that if you record a log message with error severity and have an exception caught, then it's a good practice to do not lose the exception details. You can just add "e" as the last argument to "logger.error(...)"